app/views/storage_locations/show.html.erb

```diff
@@ -161,3 +165,11 @@
   <li>Save the file as a csv file.</li>
   <li>DO NOT CHANGE ANY ITEM NAME.</li>
 <% end %>
+
+<script>
```

Comment:
I just remembered that we have https://github.com/jhund/filterrific installed which handles a bit of interactivity to change the results of a search on changes of filters. Not 100% if it is able to solve this problem, but could be good to use this gem we already have if possible.

I'am also thinking a little bit about whether the reactivity is worth it? There is a simple version of this where it is a submit form and the results get re-populated. There are pros/cons of that approach and wanted to know if you considered that approach.

Reply:
Great point! My original design was to put this field in the actual tab bar and have it apply to all tabs, but then I realized that the other 2 tabs won't work with it since they don't show InventoryItems. I then moved it into this tab but kept everything else. :) I'll add a submit button and we should be done!